compiler-plugin-tests/src/test/java/io/ballerina/stdlib/rabbitmq/plugin/RabbitmqCompilerPluginTest.java

```diff
@@ -272,19 +298,84 @@ public void testInvalidService15() {
         DiagnosticResult diagnosticResult = compilation.diagnosticResult();
         Assert.assertEquals(diagnosticResult.errors().size(), 1);
         Diagnostic diagnostic = (Diagnostic) diagnosticResult.errors().toArray()[0];
-        assertDiagnostic(diagnostic, PluginConstants.CompilationErrors.INVALID_FUNCTION);
+        assertDiagnostic(diagnostic, INVALID_FUNCTION);
     }
 
     @Test
     public void testInvalidService16() {
         Package currentPackage = loadPackage("invalid_service_16");
         PackageCompilation compilation = currentPackage.getCompilation();
         DiagnosticResult diagnosticResult = compilation.diagnosticResult();
-        Assert.assertEquals(diagnosticResult.errors().size(), 3);
+        Assert.assertEquals(diagnosticResult.errors().size(), 2);
```

Comment:
```suggestion
        Assert.assertEquals(diagnosticResult.errorCount(), 2);
```

Reply:
Let's use this in other places as well.

Reply:
Updated in https://github.com/ballerina-platform/module-ballerinax-rabbitmq/pull/393/commits/b981317fec0692ca8130a099798591997bf0bba1